src/test/java/org/folio/test/support/matcher/AccountMatchers.java

```diff
@@ -21,4 +28,30 @@ private AccountMatchers() {}
         hasJsonPath("remaining", is(0.0))
       ));
   }
+
+  public static Account verifyAccountAndGet(
+    ResourceClient accountsClient,
+    String accountId,
+    String expectedPaymentStatus,
+    float amount,
+    String statusName) {
+
+    final Response getAccountByIdResponse = accountsClient.getById(accountId);
+    getAccountByIdResponse
+      .then()
+      .body("remaining", is(amount))
+      .body("status.name", is(statusName))
+      .body("paymentStatus.name", is(expectedPaymentStatus))
+      .body("dateUpdated", notNullValue())
+      .body("metadata.updatedDate", notNullValue());
+
+    final Account updatedAccount = getAccountByIdResponse.as(Account.class);
+    final Date dateCreated = updatedAccount.getMetadata().getCreatedDate();
+    final Date dateUpdated = updatedAccount.getMetadata().getUpdatedDate();
+
+    assertThat(dateUpdated, equalTo(updatedAccount.getDateUpdated()));
+    assertThat(dateCreated, not(equalTo(dateUpdated)));
+
+    return updatedAccount;
+  }
```

Comment:
I like the idea of moving this logic to a single place and using it from different test classes, but `...Matcher` class is probably not the best place for it. Our matcher classes typically just have static methods that create JSON matchers, they don't know anything about clients or making requests etc. To me this looks like more of a `...Test` class concern. What if we create a class that extends `ApiTests`, put this method in it and make all the `...Test` classes that need this method to extend it? Or move this method to a new util class?

Reply:
Agree with you. I like the approach with `BaseTest`.